front/scripts/gerenciamento_alunos.js

```diff
@@ -134,9 +134,10 @@ async function editarAluno(RA) {
 
   const aluno_nasc = document.createElement("p");
   aluno_nasc.textContent = "data de nascimento:";
+  const data_nascimento_formatada = aluno.data_nascimento ? aluno.data_nascimento.split('/').reverse().join('-') : ''
```

Comment:
Usa a biblioteca moment pra formatar por favor

Acho que nessa página ainda não ta importada a cdn no html.
usa esse mr de base: https://github.com/ClaraSantosmf/FATEC-API-1-SEMESTRE/pull/194

Reply:
Marilia, como é para exibir na tela, não tem problema. 
O formato padrão do input type='date' é sempre yyyy-mm-dd. Se atribuir o valor, qualquer navegador compreenderá e exibirá, independente da região.
E eu deixei exatamente como a tela de editar turma, que está funcionando normalmente.

Acredito que o moment.js deve ser utilizado só para padronizar na gravação.

Reply:
Entendi caio, mas pergunta, o moment ajuda nessa formatação e substitui o split reverse e join? Pq se sim, fica melhor usar uma única forma de formatação de data, se não fica confuso o código quando a gente aborda de X maneira aqui e Y maneira ali.

Reply:
Então clara, se eu for utilizar o momentjs, vou ter que criar um Date. Para criar uma data nova, vou ter que formatar de qualquer jeito para ser recebida em um new Date(). Isso tudo pq não armazenamos o formato de data correto no banco de dados (YYYY-MM-DD). Eu deixaria passar essa issue e abriria outro débito técnico para corrigir a formatação da data no banco.

Reply:
ok então